src/Jecoute/SurveyQuestionTypeEnum.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace AppBundle\Jecoute;
+
+use MyCLabs\Enum\Enum;
+
+class SurveyQuestionTypeEnum extends Enum
+{
+    public const MULTIPLE_CHOICE_TYPE = 'multiple_choice';
+
+    public const UNIQUE_CHOICE_TYPE = 'unique_choice';
+
+    public const SIMPLE_FIELD = 'simple_field';
+
+    public const MULTIPLE_CHOICE_TYPE_LABEL = 'Choix multiples';
+
+    public const UNIQUE_CHOICE_TYPE_LABEL = 'Choix unique';
+
+    public const SIMPLE_FIELD_LABEL = 'Champ libre';
+
+    public const ALL_TYPES = [
```

Comment:
what about transform this in a method 
```
public static function all(): array
{
  return [
        self::MULTIPLE_CHOICE_TYPE_LABEL => self::MULTIPLE_CHOICE_TYPE,
        self::UNIQUE_CHOICE_TYPE_LABEL => self::UNIQUE_CHOICE_TYPE,
        self::SIMPLE_FIELD_LABEL => self::SIMPLE_FIELD,
  ];
}
```
and we can use it in the FormType with `SurveyQuestionTypeEnum::all()` call and as the callback for `Asset\Choice` `@Asset\Choice(callback={"AppBundle\Jecoute\SurveyQuestionTypeEnum", "all"})`.
and we can remove `getAllTypes` from the Entity :) 

WDYT?

Reply:
Nice idea, thx you :+1: